Containers/clamav/supervisord.conf

```
@@ -21,3 +21,10 @@ stdout_logfile_maxbytes=0
 stderr_logfile=/dev/stderr
 stderr_logfile_maxbytes=0
 command=clamd --foreground --config-file=/tmp/clamd.conf
+
+[program:milter]
+stdout_logfile=/dev/stdout
+stdout_logfile_maxbytes=0
+stderr_logfile=/dev/stderr
+stderr_logfile_maxbytes=0
+command=clamav-milter --foreground --config-file=/etc/clamav/clamav-milter.conf
```

```suggestion
command=clamav-milter --config-file=/etc/clamav/clamav-milter.conf
```
The foreground option seems to not exist.
See: https://manpages.debian.org/testing/clamav-milter/clamav-milter.8.en.html